devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy

```diff
@@ -204,8 +204,10 @@ def setLevel(value, rate = null) {
 	def additionalCmds = []
 	if (device.getDataValue("model") == "iQBR30" && value.toInteger() > 0) { // Handle iQ bulb not following spec
 		additionalCmds = zigbee.on()
-	} else if (device.getDataValue("manufacturer") == "MRVL") { // Handle marvel stack not reporting
+	} else if (isMRVL()) { // Handle marvel stack not reporting
 		additionalCmds = refresh()
+	} else if (isLeviton()) {
+		additionalCmds = zigbee.levelRefresh()
```

Comment:
as far as I know, none of this would ever have been run for a device running locally in the first place